app/src/main/java/org/mozilla/fenix/wallpapers/LegacyWallpaperMigration.kt

```diff
@@ -95,8 +95,11 @@ class LegacyWallpaperMigration(
             if (wallpaperName == TURNING_RED_MEI_WALLPAPER_NAME || wallpaperName == TURNING_RED_PANDA_WALLPAPER_NAME) {
                 settings.currentWallpaperTextColor = TURNING_RED_WALLPAPER_TEXT_COLOR.toLong(radix = 16)
             }
+
+            migrateExpiredWallpaperCardColors(wallpaperName)
```

Comment:
Could this call exist in the conditional above?

Reply:
Sure, I can relocate it